streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -212,23 +227,24 @@ public void shouldCloseActiveTasksWhenHandlingLostTasks() {
         final Task task00 = new StateMachineTask(taskId00, taskId00Partitions, true);
         final Task task01 = new StateMachineTask(taskId01, taskId01Partitions, false);
 
+        // `handleAssignment`
```

Comment:
Could we verify the assignment stack and lost stack separately, by doing `handleAssignment` verify first before calling `handleLost`

Reply:
Not sure if I can follow? The comments just mark which setup calls belongs to which test call, nothing more. All setup is done upfront before we call the actually methods under test.